docs/docs/glossary.md

```diff
@@ -0,0 +1,70 @@
+# Glossary
+
+This glossary introduces, and tries to define, the major technical terms used in the openEO project.
+
+The acronym _openEO_ contracts two concepts:
+
+- **open**: used here in the context of open source software; open source software is available in source code form, and can be freely modified and redistributed; the openEO project will create open source software, reusable under a liberal open source license (Apache 2.0)
+- **EO**: Earth observation; openEO targets the processing and analysis of Earth observation data
+
+Further terms:
+
+- **API**: application programming interface ([wikipedia](https://en.wikipedia.org/wiki/Application_programming_interface)); a communication protocol between client and back-end
+- **client**: software environment (software) that end-users directly interact with, e.g. R (rstudio), python (jupyter notebook), and javascript (web browser); R and python are two major data science platforms; javascript is a major language for web development
+- **(cloud) back-end**: server; computer infrastructure (one or more physical computers or virtual machines) used for storing EO data and processing it
+- **big Earth observation cloud back-end** server infrastructure where industry and researchers analyse large amounts of EO data
+- **simple** many end-users now use python or R to analyse data and javascript to develop web applications; analysing large amounts of EO imagery should be equally simple, and seamlessly integrate with existing workflows
+- **unified** current EO cloud back-ends all have [a different API](http://r-spatial.org/2016/11/29/openeo.html), making EO data analysis hard to validate,difficult to reproduce, and back-ends difficult to compare in terms of capability and costs, or to combine in a joint analysis across back-ends. A unified API can resolve many of these problems.
```

Comment:
"unified" isn't really explained here, instead a rundown of "what it's not" is given.
"simple" isn't really *defined* either.
IMO the two definitions should be replaced with a proper one (I can imagine that for "unified") or removed from the glossary (the meaning of "simple" really is common sense).

Reply:
Moved to issue #26 for later handling.